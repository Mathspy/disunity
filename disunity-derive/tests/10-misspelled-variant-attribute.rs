use disunity_derive::Variant;

#[derive(Variant)]
enum Class {
    #[disunity(misspelled = 3)]
    Magic(i32),
}

fn main() {}
