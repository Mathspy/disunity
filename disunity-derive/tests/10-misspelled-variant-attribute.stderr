error: expected discriminant = N inside of disunity attribute
 --> tests/10-misspelled-variant-attribute.rs:5:16
  |
5 |     #[disunity(misspelled = 3)]
  |                ^^^^^^^^^^^^^^
